### ISSUE TITLE
Callbacks from Plugins not working on mobile Chrome on Android

### ISSUE THREAD

**Issue by tinokremer** [idno/Known#1336]
## While trying to do this:
When creating a new "journal" (my own plugin) as well as a new "Checkin"
## I encountered this error:
the location is not processed and the map does not show.
## Some other notes:
I checked the error logs of the server. Nothing reported there. In the desktop browsers (Chrome, FireFox, Internet Explorer and Edge) all works fine. I reverted back to https://github.com/idno/Known/commit/62101d00ba976aff48dbba44a5a26aa0e972056b and all is fine.

**Comment by kylewm** [comment]
can you make sure `/IdnoPlugins/Checkin/checkin.js` is being loaded when you try to create a new Location post?

**Comment by tinokremer** [comment]
Hard to debug on Android's Chrome browser @kylewm . On my desktop that certainly is the case, but there it's working fine anyway.

**Comment by danito** [comment]
you're sure that you allowed your site to share your location? (one never knows)

**Comment by tinokremer** [comment]
@danito, I went back to an earlier commit (https://github.com/idno/Known/commit/62101d00ba976aff48dbba44a5a26aa0e972056b) as stated above and now it works fine. So we can be sure something happened between that commit and the latest one of this morning :)

**Comment by kylewm** [comment]
I'm on the latest code from github, installed the Journal plugin, and tested it from Chrome on Android. Both editors seems to work -- they find my location and display a map. So at a bit of a loss.

**Comment by danito** [comment]
I did an update and at first I had the same problem, it waited for ever to get location. 
I followed this [android-remote-debugging](https://developers.google.com/web/tools/chrome-devtools/debug/remote-debugging/remote-debugging to debug), but after activating (and updating Chrome on my Workstation), It worked fine... Sorry not much of a help.

**Comment by tinokremer** [comment]
I'm not beginning to wonder if something odd happened with my device.

I just grabbed the latest (commit https://github.com/idno/Known/commit/bd51ca7c8a2c5c47b1a95f30c27acb6eef8855ff) code from github and installed. I now encountered the same behavior as @danito, the first time it took a long time (I did a test before on the old code and it came instantly). Then I tried it again and it's working normally again. I'll do some more testing, closing this issue in the meantime. I think this may be a false report from me. Not sure what happened yesterday, but likely it wasn't Known causing issues.

Sorry for wasting your time, thanks for your efforts!
